ui_sim.py

```diff
@@ -247,3 +261,16 @@ def runXTurns(self):
 
     def displayPoints(self):
         self.sim.getPointsData()
+
+    def displayScoreboard(self):
```

Comment:
Green team won in this instance by collecting the object. Should score maybe be 1 to reflect this?
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/e49144c2-37c7-4b6d-8ee3-b0e7bfbc3487)

Reply:
scoring functionality hasn't actually been updated at all (as in only damage inflicted counts as points). i think at this point we are marking this as a limitation within the system and leaving this for future work.